python/ray/serve/_private/api.py

```diff
@@ -329,3 +334,23 @@ def serve_start(
         f'namespace "{SERVE_NAMESPACE}".'
     )
     return client
+
+def call_app_builder_with_args_if_necessary(builder: Union[ClassNode, FunctionNode, FunctionType], args_dict: Dict[str, Any]) -> Union[ClassNode, FunctionNode]:
```

Comment:
is `Any` here for future use case? I thought we only have string.

Reply:
I realized while writing this that in the config file case users can pass non-string values, e.g.,
```yaml
args:
  - model_uri: "some_string"
    model_num_replicas: 10
```

the `10` will be passed as an integer